app/assets/javascripts/admin/templates/modal/admin-install-theme.hbs

```diff
@@ -0,0 +1,122 @@
+      {{d-icon 'caret-right'}}
+    </div>
+    <div class="install-theme-item">
+      {{radio-button name="upload" id="create" value="create" selection=selection}}
+      <label class="radio" for="create">{{d-icon 'plus'}} {{i18n 'admin.customize.theme.install_create'}}</label>
+      {{d-icon 'caret-right'}}
+    </div>
+  </div>
+  <div class="install-theme-content">
+    {{#if popular}}
+      <div class="popular-theme-items">
+        {{#each themes as |theme|}}
+          <div class="popular-theme-item">
+            <div class="popular-theme-name">
+              <div class="popular-theme-type">
+                {{#if theme.component}}
+                  {{d-icon 'puzzle-piece'}} {{i18n 'admin.customize.theme.component'}}
+                {{else}}
+                  {{i18n 'admin.customize.theme.theme'}}
+                {{/if}}
+              </div>
+              <a href="{{theme.meta_url}}" target="_blank">{{theme.name}}</a>
+            </div>
+            <div class="popular-theme-buttons">
+              {{#if theme.installed}}
+                <span>{{I18n "admin.customize.theme.installed"}}</span>
+              {{else}}
+                {{d-button class='btn-small'
+                  label="admin.customize.theme.install"
+                  disabled=installDisabled
+                  icon="upload"
+                  action=(action "installThemeFromList" theme.value)}}
+
+                {{#if theme.preview}}
+                  <a href="{{theme.preview}}" target="_blank">{{d-icon "desktop"}} {{I18n "admin.customize.theme.preview"}}</a>
+                {{/if}}
+              {{/if}}
+            </div>
+          </div>
+        {{/each}}
+      </div>
+    {{/if}}
+
+    {{#if local}}
+      <div class="inputs">
+        <input onchange={{action "uploadLocaleFile"}} type="file" id="file-input" accept='.dcstyle.json,application/json,.tar.gz,application/x-gzip'><br>
+        <span class="description">{{i18n 'admin.customize.theme.import_file_tip'}}</span>
+      </div>
+    {{/if}}
+
+    {{#if remote}}
+      <div class="inputs">
+        <div class='repo'>
+          <div class="label">{{i18n 'admin.customize.theme.import_web_tip'}}</div>
+          {{input value=uploadUrl placeholder=urlPlaceholder}}
+        </div>
+
+        {{d-button
+          class="btn-small advanced-repo"
+          action=(action "toggleAdvanced")
+          label='admin.customize.theme.import_web_advanced'}}
+
+        {{#if advancedVisible}}
+          <div class='branch'>
+            <div class="label">{{i18n 'admin.customize.theme.remote_branch'}}</div>
+            {{input value=branch placeholder="master"}}
+          </div>
+
+          <div class='check-private'>
+            <label>
+              {{input type="checkbox" checked=privateChecked}}
+              {{i18n 'admin.customize.theme.is_private'}}
+            </label>
+          </div>
+          {{#if checkPrivate}}
```

Comment:
This is a lot of nested `if` statements. It would be cleaner to create a new computed property for this.